module/library/individualAssessment/validator/v1.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * name : v1.js
+ * author : Aman
+ * created-date : 04-Jun-2020
+ * Description : Individual solution validation.
```

Comment:
@aman-tunerlabs - Description has to be "Individual assessment solution validation file"